base/applications/network/netstat/netstat.c

```diff
@@ -98,6 +98,9 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                         // UNIMPLEMENTED.
                         ConPuts(StdErr, L"'b' option is FIXME (Accepted option though unimplemented feature).\n");
                         bDoShowProcName = TRUE;
+#if _WIN32_WINNT < 0x600
```

Comment:
```suggestion
#if _WIN32_WINNT < _WIN32_WINNT_VISTA
```

Reply:
`#if (_WIN32_WINNT < _WIN32_WINNT_VISTA)`
even though I would prefer, moe like Joachim, that this behaviour stays even when we compile ROS in Vista+ mode.

Reply:
Applied